src/VisualStudio/IntegrationTest/IntegrationTests/Workspace/WorkspaceBase.cs

```diff
@@ -113,5 +113,20 @@ public void RenamingOpenFiles()
             // ...and after.
             Assert.Contains(ProjectName, VisualStudio.Editor.GetProjectNavBarItems());
         }
+
+        [WpfFact]
+        public virtual void RenamingOpenFilesViaDTE()
```

Comment:
Why are we keeping both rename tests? Roslyn doesn't care about how renames happen, it's the project system handling all of that anyways?

Reply:
Or put another way: if one rename works and the other doesn't, that's either a Roslyn bug or a project system bug. We need to investigate that.

Reply:
These are integration tests, with a demonstrated case that the method of renaming matters in integration scenarios.

Reply:
Sure, but they should be in the project system team's integration tests, no?

Reply:
The lines are blurred because the test requires correct behavior of code from many different teams in order to pass. I have a hard time saying any particular team is better than any other.